### ISSUE TITLE
Can you add a screenshot to the README.md?

### ISSUE THREAD

**Issue by stevleibelt** [Jirubizu/localai-admin#2]
Hello,

as an enhancement, I would like you to add a screenshot of the UI to the README.md.

From my point of view, there is a higher chance I even give a UI project a try when I could see upfront how it looks like.

Best regards and thanks for your work so far.

**Comment by gabfr** [comment]
true

**Comment by lee-b** [comment]
Agree, documentation is lacking - it's not at all clear what this is, whether it's up to date, official, semi-official, whether it was a good idea at one time but now LocalAI has the same features implemented in a better way, whether this WAS/IS the official project to develop those features but they got/get merged into LocalAI on a regular basis, etc.  It might be great, but a screenshot would be a bare minimum to even consider using this, to be honest.
